### PR TITLE
Issue #6848: add xpath regression test for UnnecessarySemicolonInEnum…

### PR DESCRIPTION
**Pull request by strkkk**
#6848

### CODE REVIEW

**Comment by strkkk** [comment]
pitest failure is not related with code changes in PR

**Comment by romani** [comment]
cirlcleci is relaunched.

Travis failure is stable and unrelated.
looks like we need to follow https://github.com/sormuras/bach/issues/51#issuecomment-487382675
Google people looks like resolved similar issue as suggested in this comment, their commit - https://github.com/google/truth/commit/8f33bf6015daaaec06391da7cc700f20b28ef935

@pbludov , what do you think? should we do the same? define "dist" in travis.yml

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUnnecessarySemicolonInEnumerationTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.junit.Test;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.checks.coding.UnnecessarySemicolonInEnumerationCheck;
+
+public class XpathRegressionUnnecessarySemicolonInEnumerationTest
+        extends AbstractXpathTestSupport {
+
+    private final String checkName =
+            UnnecessarySemicolonInEnumerationCheck.class.getSimpleName();
```
Comment (romani) [inline src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUnnecessarySemicolonInEnumerationTest.java]: please add your Check to https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_filters.xml#L670

I hope one day we will be able to enforce in CI to notify on what should be done, and eventually remove this list of Checks, as all Checks become xpath ready.

Comment (strkkk) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUnnecessarySemicolonInEnumerationTest.java]: done

**Comment by pbludov** [comment]
> Travis failure is stable and unrelated.
> looks like we need to follow [sormuras/bach#51 (comment)](https://github.com/sormuras/bach/issues/51#issuecomment-487382675)
> Google people looks like resolved similar issue as suggested in this comment, their commit - [google/truth@8f33bf6](https://github.com/google/truth/commit/8f33bf6015daaaec06391da7cc700f20b28ef935)
> 
> @pbludov , what do you think? should we do the same? define "dist" in travis.yml

For now we can only drop OracleJDK builds from the Travis CI completely.
Maybe we should set up a test environment for OracleJDK with official docker [image](https://hub.docker.com/publishers/oracle)

**Comment by romani** [comment]
Let's drop oraclejdk from Travis for now

**Pull request merged**